### DISCUSSION TITLE
Can I use a texture from one application in another application?

### DISCUSSION THREAD

**Discussion by dbuzinin** [pixijs/pixijs#9081] [Q&A]
There are 2 pixi applications on the page. One is the main one, the other as a mini-map. In the main application I create Sprite by drawing complex shapes through graphics and generating textures. 

Is there any way to take the resulting textures from one app and create sprites in the other?

So far I'm not getting anywhere. I get the textures, but the sprites created with them don't show up in the second app even though they are added to it.

**Comment by SuperSodaSea** [comment]
Perhaps you can try [`PIXI.Extract`](https://pixijs.download/dev/docs/PIXI.Extract.html).

Note that the examples in the documentation may be somewhat outdated, for PixiJS v7 the code is like this:

```js
const image = await app.renderer.extract.image(graphics);
```

Then use `image` to create a new `Texture` for another `PIXI.Application`.
